src/FormBuilder/resources/js/form-builder/src/styles/_components.scss

```diff
@@ -208,10 +208,12 @@ This creates a consistent separator between the tabs and the rest of the sidebar
     border-color: black;
 }
 
-.components-checkbox-control__label {
-    font-size: 1rem;
-    font-weight: 500;
-    color: #595959;
+.give-terms-and-conditions-control {
+    .components-checkbox-control__label {
+        font-size: 1rem;
+        font-weight: 500;
+        color: #595959;
+    }
 }
```

Comment:
Lets actually move this to live inside the terms block folder and just import it into the `Edit.tsx` file.  This is my preferred way of handling component styles with our build process - because importing styles ends up producing the same outcome (one stylesheet) and then the styles get to live alongside their implementation which is easier to see/find.  

I realize our current styles folder conflicts with that mentality but that's the direction I would like to move us toward.

Reply:
Awesome that makes sense. When things start to cool down, I'll work on migrating things into there directories & out of inline as well.